FlagsmithClient/src/main/java/com/flagsmith/Flagsmith.kt

```diff
@@ -50,8 +109,12 @@ class Flagsmith constructor(
     }
 
     fun getFeatureFlags(identity: String? = null, result: (Result<List<Flag>>) -> Unit) {
+        // Save the last used identity as we'll refresh with this if we get update events
+        lastUsedIdentity = identity
+
         if (identity != null) {
             retrofit.getIdentityFlagsAndTraits(identity).enqueueWithResult { res ->
+                flagUpdateFlow.tryEmit(res.getOrNull()?.flags ?: emptyList())
```

Comment:
I'm not sure on the logic behind emitting the event here? Is this so that in the multiprocess android world, another event listener can pick up when another process updates the flags? I guess it's questionable whether this is the desired functionality. What was the thought process here?

Reply:
It's so that the Android viewmodels can pick up when the flags are updated in a reactive way, and is a standard paradigm in Android.

This is, AFAIK, analogous to the onChange you have in the JS client which I was told to copy.